src/test/java/org/commcare/formplayer/tests/BaseTestClass.java

```diff
@@ -1130,6 +1163,16 @@ private <T> T generateMockQuery(ControllerType controllerType,
                 break;
         }
         switch (requestType) {
+            case MULTIPART:
+                MockMultipartFile answer = new MockMultipartFile("answer", "answer",
+                        "application/json", ((String)bean).getBytes());
+                result = controller.perform(
+                        multipart(urlPrepend(urlPath))
+                                .file(file)
+                                .file(answer)
```

Comment:
this shouldn't be a file, it should just be a normal 'part':
```suggestion
                                .part(new MockPart("answer",  ((String)bean).getBytes()))
```

Reply:
https://github.com/dimagi/formplayer/pull/1198/commits/36ca13ffb061768da7237041d4e961d6b49f2480